.github/workflows/build.yaml

```
@@ -132,7 +132,8 @@ jobs:
           build-args: |-
             NODE_VERSION=18
           platforms: ${{ env.CONTAINER_ARCHS }}
-          push: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' && github.repository == 'opentdf/frontend' }}
+          # publish the image if there is a push event to main or a release event
+          push: ${{ github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.event.release) && github.repository == 'opentdf/frontend' }}
```

event_name will be 'release' if there is a release, I think, so you need to regroup these conditions. I'm pretty sure the GitHub.ref will be whatever tag is in your release but I'm not sure - either way, it is covered by the `is-ancestor` condition on line 68 so it is safe to not check it on a release event.

```suggestion
          push: ${{ ((github.event_name == 'push' && github.ref == 'refs/heads/main') || github.event.release) && github.repository == 'opentdf/frontend' }}
```